third_party/imgui/BUILD.gn

```diff
@@ -23,8 +23,19 @@ config("imgui_config") {
   defines = [ "CHIP_IMGUI_ENABLED=1" ]
 
   # FUTURE: these should be parsed from `sdl2-config --cflags`
-  include_dirs += [ "/usr/include/SDL2" ]
-  defines += [ "_REENTRANT" ]
+  if (current_os == "mac") {
+    # Default path setup by `brew install sdl2`
+    include_dirs += [ "/usr/local/include/SDL2" ]
```

Comment:
What if the system is using macports instead of brew?

Followup: this should be using pkgconfig or gn args or something to get this path.  I guess you have the FUTURE bit above already, so that might be enough....

Reply:
the correct thing is to use `sdl-config` which would get:

```
sdl2-config --cflags
-I/usr/local/include/SDL2 -D_THREAD_SAFE
```

Which would be doable but requires a python bit that invokes it (to integrate with gn) and then parses (to split includes and defines) and the update things.

I kept the FUTURE/TODO because this seems reasonably tedious for something that is 'local test UI' only - I never expect real products to actually use imgui/sdl2/opengl from chip.